scripts/build/builders/imx.py

```diff
@@ -74,30 +75,58 @@ def __init__(self,
         self.release = release
         self.app = app
 
-    def GnBuildEnv(self):
-        return {
-            'PKG_CONFIG_PATH': self.SysRootPath('IMX_SDK_ROOT') + '/sysroots/cortexa53-crypto-poky-linux/lib/aarch64-linux-gnu/pkgconfig',
-        }
-
     def GnBuildArgs(self):
+        entries = os.listdir(self.SysRootPath('IMX_SDK_ROOT'))
+        for entry in entries:
+            if re.match(r'^environment-setup-', entry):
+                env_setup_script = entry
+                break
+
```

Comment:
should we throw an exception if env_setup_script is None? 

Also may need extra checks to ensure that lines for all of CC, CXX and such are found.

Reply:
exceptions are thrown when the env setup script is not found or related envs are not found in the env setup script now.